### ISSUE TITLE
Support for forked repositories is compliacted and does not work for me

### ISSUE THREAD

**Issue by rugk** [EnricoMi/publish-unit-test-result-action#597]
I tried following https://github.com/marketplace/actions/publish-test-results#support-fork-repositories-and-dependabot-branches, but it did not work.

As soon as you try to download it with a specific run ID, it cannot find it: https://github.com/PrivateBin/PrivateBin/actions/runs/8950164072/job/24585250115?pr=1303

What works:
* Using the official "download artifact" action works, but it seems it cannot find the "event file" suggested: https://github.com/PrivateBin/PrivateBin/actions/runs/8950131683/job/24585173950?pr=1303
* Also before trying to integrate that "contributor support", it worked. (Without zthis event file etc.)
* I use a test matrix.

PR for reference: https://github.com/PrivateBin/PrivateBin/pull/1303

**Comment by EnricoMi** [comment]
In step 4, the instructions say "Add the following workflow that publishes test results". This refers to a **separate** workflow. So in stead of adding the publish action to the `Tests` workflow, please create a new workflow file with the given example YAML.

**Comment by rugk** [comment]
Ah thanks a lot! That said, I changed it, but GitHub does not trigger the "Test results" workflow at all – and yes, obviously I've changed the name.

https://github.com/PrivateBin/PrivateBin/pull/1303

Possibly because the old workflow was cancelled (although that's also not intended, but that's a different issue, but [the GitHub doc](https://docs.github.com/de/actions/using-workflows/events-that-trigger-workflows) does not actually mention any "cancelled" or so event to be added.
And after all, for all test runs that have completed (even if not all of them have), results should be available and be uploaded. Again [the doc says it does not depend on whether it was success or failure](https://docs.github.com/de/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow) though…
